submissions/cynthiaNwakaeme/CSS/product.css

```diff
@@ -0,0 +1,255 @@
+header nav{
+    float: right;
+    margin-top: 15px;
+}
+header a:hover{
+    text-transform: uppercase;
+    color:white;
+    font-weight: bold;
+
+}
+/* THE CONTAINER */
+main{
+    width: 90%;
+    padding: 2%;
+    overflow: hidden;
+    
+}
+/* MAIN */
+.main-img{
+    float: left;
+    width: 230px;
+    margin: 10px 0 10px 30px;   
+    border: 1px solid rgb(129, 129, 129);    
+}
+.main-img img{
+    width: 85%;
+    padding: 2% 10%;
+    height: inherit;
+}
+        /* MAIN IMAGE DETAILS */
+.img-details{
+    float:left;
+    width: 305px; 
+    margin-left: 30px;
+}
+.img-details h3,
+.img-details h4
+{
+    padding: 0;
+    margin: 13px 0;
+}
+#quality h4, #quality table{
+    display: inline-table;
+    padding: 0 ;
+    margin: 0 ;
+
+}
+#quality table{
+    border-collapse: collapse;
+}
+#quality table,#quality th{
+    border: 1px solid rgb(129, 129, 129);
+    padding: 0 8px;
+}
+#quality tr th a{
+    text-decoration: none;
+}
+.img-details button{
+    width: 130px;
+    border-radius: 12px;
+    background-color: goldenrod;
+    color: #fff;
+    padding: 10px;
+    border: none;
+    font-size: 16px;
+}
+.description{
+    margin:0 48px; 
+    padding-bottom: 20px;
+}
+.description h4{
+    padding: 0;
+    margin: 0;
+}
+.description p{
+    padding: 0;
+    margin: 8px 0;
+}
+/* ASIDE: THIS IS FOR RELATED PRODUCT */
+aside{
+    background-color: rgb(231, 227, 227);
+    padding:4% 0 5% 0;
+    margin: auto;
+    overflow: hidden;
+}
+aside h3{
+    padding: 0;
+    margin: 0;
+    text-align: center;
+}
+.img-slide{
+    float: left;
+    width: 30%;
+    height: 10%;
+    margin:20px 0 0 30px;
+    background-color: white;
+}
+.img-slide a img{
+    width: 65%;
+    height: 20%;
+    margin:10px 60px 0 60px;
+}
+.img-slide p{
+    text-align: center;
+    font-weight: bold;
+    margin-bottom: 0 ;
+}
+
+/* THE FOOTER */
+footer{
+    overflow: hidden;
+    background-color: #9d426b;
+    padding:20px 0 0 50px;
+    color: white;
+}
+.footer-box,
+.footer-box-2,
+.footer-box-3{
+    float: left;
+    width: 33%;
+}
+.footer-box ul li,
+.footer-box-3 ul li
+{
+    list-style: none;
+    margin-bottom: 8px;
+}
+.footer-box ul li a{
+    text-decoration: none;
+    display: inline;
+    color: white;  
+}
+.footer-box-2 button[type="submit"]{
+    height: 30px;
+    padding: 0 5%;
+    font-size: 12px;
+    color: #9d426b;
+    font-weight: bold;
+}
+.footer-box-2 input[type="email"]{
+    height: 20px;
+    padding: 3px 18px 3px ;
+    font-size: 11px; 
+}
+.footer-box-2 input::placeholder{
+    color: #9d426b;
+}
+.footer-box-3 h5{
+    padding: 0;
+    margin: 0;
+}
+.footer-box-3 ul li{
+    display: inline-flex;
+    padding: 8px ;
+}
+.footer-box-3 ul li img{
+    width: 35px;
+}
+footer p{
+    display: block;
+}
+/* HANDLING RESPONSIVENESS / RESOLUTIONS */
+@media screen and (max-width:595px){
+    .main-img{
+        padding: 0 30px;
+    }
+    aside{
+        padding-bottom: 50px;
+    }
+    aside .img-slide{
```

Comment:
Is it possible that this class is used only once in that file?, then no need to call `aside` with it.

Reply:
You can leave it as is then.

Reply:
Thank you so much